plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/BaseIcebergConnectorSmokeTest.java

```diff
@@ -484,6 +496,114 @@ public void testFileSortingWithLargerTable()
         }
     }
 
+    @Test
+    public void testDropTableWithMissingMetadataFile()
```

Comment:
Sorry, I could not understand the comment. Could you please explain a bit?

>the connector test.

Do you mean `BaseConnectorSmokeTest.java`?

Reply:
I guess it means `BaseConnectorTest`.

Reply:
My purpose for adding the test cases in smoke test is because it covers all catalogs. whereas `BaseIcebergConnectorTest` only covers `HiveCatalog` and `GlueCatalog`.

> should generally have <= coverage

In this case, should the tests be duplicated in `BaseIcebergConnectorTest`?